src/gsy_e/models/strategy/load_hours.py

```diff
@@ -259,11 +254,8 @@ def area_reconfigure_event(self, *args, **kwargs):
 
     def event_activate_price(self):
         """Update the strategy prices upon the activation and validate them afterwards."""
-        # If use_market_maker_rate is true, overwrite final_buying_rate to market maker rate
         if self.use_market_maker_rate:
```

Comment:
Why not including the `use_market_maker_rate` check in the mixin too ? That way it might be easier in the future if we revert from inheritance to composition (convert the mixin to a normal class that will be constructed in the LoadHoursStrategy constructor)

Reply:
Makes sense, will do.